pkg/plugin/volume_snapshotter_plugin.go

```diff
@@ -111,63 +130,22 @@ func (p *NewVolumeSnapshotter) IsVolumeReady(volumeID, volumeAZ string) (ready b
 // set of tags to the snapshot.
 func (p *NewVolumeSnapshotter) CreateSnapshot(volumeID, volumeAZ string, tags map[string]string) (string, error) {
 	p.Infof("CreateSnapshot called with volumeID %s, volumeAZ %s, tags %v", volumeID, volumeAZ, tags)
-	var snapshotID string
-
-	// call SnapshotMgr CreateSnapshot API
-	peID := astrolabe.NewProtectedEntityID("ivd", volumeID)
-	peID, err := p.snapMgr.CreateSnapshot(peID, tags)
-	if err != nil {
-		p.WithError(err).Errorf("Fail at calling SnapshotManager CreateSnapshot from peID %v, tags %v", peID, tags)
-		return "", err
-	}
-
-	// Construct the snapshotID for cns volume
-	snapshotID = peID.String()
-	p.Debugf("The snapshotID depends on the Astrolabe PE ID in the format, <peType>:<id>:<snapshotID>, %s", snapshotID)
-
-	p.Infof("CreateSnapshot completed with snapshotID, %s", snapshotID)
-	return snapshotID, nil
+	p.Warnf("Snapshot operation not supported on volume snapshotter plugin")
+	return "", nil
 }
 
 // DeleteSnapshot deletes the specified volume snapshot.
 func (p *NewVolumeSnapshotter) DeleteSnapshot(snapshotID string) error {
-	p.Infof("DeleteSnapshot called with snapshotID %s", snapshotID)
-	peID, err := astrolabe.NewProtectedEntityIDFromString(snapshotID)
-	if err != nil {
-		p.WithError(err).Errorf("Fail to construct new Protected Entity ID from string %s", snapshotID)
-		return err
-	}
-
-	err = p.snapMgr.DeleteSnapshot(peID)
-	if err != nil {
-		p.WithError(err).Errorf("Failed at calling SnapshotManager DeleteSnapshot for peID %v", peID)
-		return err
-	}
-
+	p.Infof("VolumeSnapshotter: DeleteSnapshot called with snapshotID %s, "+
```

Comment:
Can we somehow keep the logging of both CreateSnapshot and DeleteSnapshot aligned?